### DISCUSSION TITLE
TreeSelectModule: The 'count' of children is showing in the label of option select

### DISCUSSION THREAD

**Discussion by hilmanmanang** [primefaces/community#715] [PrimeNG]
Hi,

I have an issue, there is a TreeNode data that I pass as [options] props in p-treeSelect component. However, the label showing the count of children. In image below, showing the Data Parent have extra "3" in the end of label because it has 3 children.

![Screenshot 2023-12-18 at 9 41 02 AM](https://github.com/primefaces/community/assets/59623098/d26cbf16-7348-4ebd-96d9-79116ec2d095)

Below is the [options] value that I passed.

<img width="370" alt="image" src="https://github.com/primefaces/community/assets/59623098/d814e015-0767-4b29-a3b7-78c87cf780b1">

When I inspect the element, I dont know where the number 3 is coming from 

![image](https://github.com/primefaces/community/assets/59623098/8936445b-3492-4c8f-8281-fd2654aafa63)
 
The 'count' below show based on number of children exist, in this case, I search 'one', It become 'Data Parent 1' because result of children's element now is one.

![image](https://github.com/primefaces/community/assets/59623098/f2747a7d-3d33-4c34-aa50-d5464b033c08)

Im using  
primeng: 16.9.1
angular: 16.2.12

When I downgrade to primeng version 15, it is working as expected.

Any advice? Thank you very much.

**Comment by cetincakiroglu** [comment]
Could you please share a stackblitz example so we can identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
